scripts/topology_generator/bottleneck.py

```diff
@@ -159,11 +162,13 @@ def generate_simulation(
             flow_name = f"flow_{flow_id}"
             flow_id += 1
             simulation["flows"][flow_name] = {
+                "type": algorithm,
```

Comment:
Lets rename `algorithm` to `flow_type`:
```suggestion
                "type": flow_type,
```